omicsclaw/common/report.py

```
@@ -353,41 +354,45 @@ def write_replot_hint(
+
+        envelope["replot"] = {
+            "available": True,
+            "command": f"python omicsclaw.py replot {skill_alias} --output {output_dir}",
+            "renderers": renderer_info,
+        }
 
-    result_path.write_text(json.dumps(envelope, indent=2, default=str))
+        result_path.write_text(json.dumps(envelope, indent=2, default=str))
+    except Exception:
+        # write_replot_hint is informational only — never crash the skill over it
+        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)